agent/engine/task_manager.go

```diff
@@ -1618,6 +1615,12 @@ func (mtask *managedTask) unstageVolumeWithRetriesAndTimeout(csiClient csiclient
 	derivedCtx, cancel := context.WithTimeout(mtask.ctx, unstageVolumeTimeout)
 	defer cancel()
 	backoff := retry.NewExponentialBackoff(unstageBackoffMin, unstageBackoffMax, unstageBackoffJitter, unstageBackoffMultiple)
+	now := time.Now()
+	defer func() {
+		logger.Debug("NodeUnstageVolume duration stats:", logger.Fields{
+			"duration": strconv.FormatFloat(time.Since(now).Seconds(), 'f', -1, 64),
```

Comment:
why use strconv.FormatFloat here but not above in watcher.go?

Reply:
Seconds() returns a float: https://pkg.go.dev/time#Duration.Seconds

Reply:
Updated this in [92b4941](https://github.com/aws/amazon-ecs-agent/pull/4326/commits/92b4941fe4a9b7eb6480f6eb9cf4c18565dd99d9)